v2/src/components/VCard.vue

```diff
@@ -246,18 +246,14 @@ const hasDetails = computed(() => {
     align-self: flex-start;
     flex: 1;
     padding: spacing(3) spacing(6) spacing(6) spacing(3);
-    overflow: hidden;
-    .card-tags {
-      display: flex;
-      flex-direction: row;
-      flex-wrap: wrap;
-      gap: spacing(2);
-      margin-bottom: spacing(2);
-    }
     .card-title {
+      line-height: var(--font-size-8);
+      .v-tag {
+        float: left;
+        margin-right: spacing(2);
+        line-height: var(--font-size-8);
+      }
       .card-title-link {
-        display: flex;
```

Comment:
add styles to card-title-link 

`
display: inline-block;
 width: 100%;
`

This will force the title to be on it's own line, and you can overwrite it to be inline for the Gothamist theme